extensions/sql-database-projects/package.json

```diff
@@ -246,6 +250,10 @@
         {
           "command": "sqlDatabaseProjects.changeTargetPlatform",
           "when": "false"
+        },
+        {
+          "command": "sqlDatabaseProjects.insertSqlInputBinding",
+          "when": "editorLangId == csharp"
```

Comment:
This could show up in ADS as well - is that intended? You should use the azdataAvailable context value if not

Reply:
ah no it should only be in vscode for now. I added the !azdataAvailable, but it wasn't showing in ADS even though I had the C# extension installed since ADS doesn't seem to recognize C# files even with the extension (C# files are still considered plaintext) :(